src/libecalc/dto/utils/aggregators.py

```diff
@@ -48,7 +48,7 @@ def aggregate_emissions(
         emissions_aggregated[emission_name] = PartialEmissionResult(
             name=emission_name,
             timesteps=emissions[0].timesteps,
-            rate=reduce(operator.add, [emission.rate for emission in emissions]),
+            rate=reduce(operator.add, [emission.rate.to_calendar_day() for emission in emissions]),
```

Comment:
here the rate should already be calendar day I guess, but it doesn hurt to be sure at least ;)

Reply:
When I checked (I think), it seemed to be stream:-)